factory.py

```diff
@@ -39,7 +39,7 @@ def create_from_verification_data(verification_data: VerificationData) -> arshin
             verification.applicable.sign_pass = verification.applicable.sign_mi = False
 
         verification.metrologist = verification_data.metrologist
-        verification.means = MeansFactory.create_test(verification_data.test_dev_num)
+        verification.means = MeansFactory.create_test(verification_data.test_dev_num, config.company_config['test_devices'])
```

Comment:
By the way, I'd implement configuration in a different way.

1. Parse the configuration in a `config` component and make it available as an hierarchy of object(s).
2. Inject that configuration (or required parts of it) into specific services while initializing them instead of requesting that configuration in the runtime inside those services.